### ISSUE TITLE
use_express_redis_cache only disables storing to cache

### ISSUE THREAD

**Issue by bkniffler** [rv-kip/express-redis-cache#50]
Hey, I noticed res.use_express_redis_cache only disables storing to the cache, but not getting from it. Wouldn't it make sense to disable read and write?

I'm looking for a solution to completely disable caching if in debug.

**Comment by woubuc** [comment]
@bkniffler if it helps, this is how I implemented a `?nocache` query parameter to disable caching.

``` Coffeescript
# Conditional caching middleware
cached = cache.route()
cacheIf = (req, res, next) ->
  if req.query.nocache?
    do next
  else
    cached req, res, next

# Get item
router.get '/v2/:id', cacheIf, (req, res) ->
  ...
```

**Comment by bkniffler** [comment]
Thanks @woubuc, I came to use another workaround

``` javascript
app['redis-cache'] = !DEBUG ? require('express-redis-cache')({
      host: 'xxx',
      port: xxx,
      auth_pass: 'xxx',
      expire: 60*60*24
   }) : {
      route: function(){
         return function(req, res, next){next()}
      }
   };
app.get('/abcabc', app['redis-cache'].route(), function (req, res)  {
      setTimeout(function(){
         res.json({hello:'world'})
      }, 5000);
   });

```

I couldn't think of one reason we would need such workarounds. Why not implement this behaviour into the core?

**Comment by woubuc** [comment]
@bkniffler I think that's what PR #49 does. It's just a matter of waiting for @rv-kip to merge it. Either that or fork the repo and implement the fix in your fork.

**Comment by rv-kip** [comment]
Fixed in v0.2.0
